src/data/entities/tester.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+    Column,
+    CreateDateColumn,
+    Entity,
+    PrimaryGeneratedColumn
+} from "typeorm";
+
+@Entity("testers")
+export class Tester {
+    @PrimaryGeneratedColumn()
+    testerId: number;
+
+    @Column({ unique: true })
+    email: string;
+
+    @Column({ unique: false })
```

Comment:
unqiue is false by default
i think its clearer to only put unique: true 
and leave others as default false value